source/index.slim

```diff
@@ -6,3 +6,4 @@ title: Mirror Conf
 = partial "index/hero"
 = partial "index/about"
 = partial "index/speakers"
+= partial "index/screening"
```

Comment:
in the meantime, I added the tickets section here as well, so please check what the correct order is for these 2